nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
Also worth noting, the current code doesn't allow n_cuts to be a float, because range needs an integer.

I guess if we expect our user to be able to use range (I hope we do), we could expect them to provide an int as n_cuts, and provide a helpful error message to guide them if they don't.